plugins/nf-azure/src/main/nextflow/cloud/azure/batch/AzBatchService.groovy

```diff
@@ -339,6 +339,13 @@ class AzBatchService implements Closeable {
 
         final resFiles = new ArrayList(10)
 
+        if( config.batch().installAzCopy!=Boolean.FALSE && !config.batch().canCreatePool() ) {
+            log.debug "[AZURE BATCH] installing azcopy as task resource"
+            resFiles << new ResourceFile()
+                    .withHttpUrl('https://nf-xpack.s3-eu-west-1.amazonaws.com/azcopy/linux_amd64_10.8.0/azcopy')
```

Comment:
Hi @jordeu ,

I was wondering whether there's a way we could make sure that we're always using the latest release?

Reply:
The opposite, silent updates break your code.

Reply:
I also think that is better to update this more manually. I am watching the releases [here](https://github.com/Azure/azure-storage-azcopy/releases) to get notified when there is a new release.